packages/muelu/src/Transfers/GeneralGeometric/MueLu_GeometricInterpolationPFactory_def.hpp

```diff
@@ -159,6 +160,10 @@ namespace MueLu {
       coarseCoordinates->replaceMap(coarseCoordsMap);
 
       Set(coarseLevel, "Coordinates", coarseCoordinates);
+
+      if(pL.get<bool>("keep coarse coords")) {
+        coarseLevel.Set<RCP<realvaluedmultivector_type> >("Coordinates2", coarseCoordinates, NoFactory::get());
```

Comment:
@lucbv 👍 Doesn't look to complicated now. One could be wondering, why we weren't able to keep coordinates during one of our earlier attempts. 🤣

Reply:
Yes once I looked at it afterwards I thought it was rather easy, of course once you have learned how things work everything looks easy. It did take some trial and error to arrive to that!